/*
 * gRPC service to receive device testing results
 */

syntax = "proto3";

import "forch/proto/devices_state.proto";
import "forch/proto/shared_constants.proto";

/*
 * gRPC service to receive testing results
 */
service DeviceReport {
    rpc ReportDevicesState (DevicesState) returns (Empty) {}
}
